src/rsz/test/repair_fanout7.ok

```diff
@@ -8,9 +8,9 @@ load DFF_X1 CK D 300000 metal3 1000
 [INFO ODB-0133]     Created 155 nets and 459 connections.
 worst slack -23.63
 [INFO RSZ-0094] Found 150 endpoints with setup violations.
-[INFO RSZ-0045] Inserted 78 buffers, 1 to split loads.
-[INFO RSZ-0041] Resized 527 instances.
-[INFO RSZ-0049] Cloned 7 instances.
+[INFO RSZ-0045] Inserted 87 buffers, 1 to split loads.
+[INFO RSZ-0041] Resized 522 instances.
+[INFO RSZ-0049] Cloned 8 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
 Repair timing output passed/skipped equivalence test
-worst slack -3.75
+worst slack -4.39
```

Comment:
Why do the results change if this is only for manual buffer removal?  If this is expected then a secure CI is needed.

Reply:
The reason for this QoR change is the newly added para update after each buffer removal.  Buffer removal is triggered after some setup buffering during "journal restore" step to recover timing.  This is the right thing to do even though this caused worse QoR for this test.